pkg/master/master.go

```diff
@@ -355,6 +356,7 @@ func (c completedConfig) New(delegationTarget genericapiserver.DelegationTarget)
 		appsrest.RESTStorageProvider{},
 		admissionregistrationrest.RESTStorageProvider{},
 		eventsrest.RESTStorageProvider{TTL: c.ExtraConfig.EventTTL},
+		psprest.RESTStorageProvider{},
```

Comment:
I had this code in the first version, but after Jordan's comment I rewrote PR from scratch and remove it as I thought that we don't need it because cohabitation should do that..

Reply:
Cohabitation overrides how group resources are stored in etcd (i.e. it puts them in the same location).  It does not do anything with the REST endpoints.